### PR TITLE
Building a GeoDataFrame from DataFrame

### LINKED ISSUE #689: DOC: add page on how to construct GeoDataFrame from existing DataFrame
Eg if you have 

- two columns with x, y
- a column with the WKT format

### PR DESCRIPTION
**Pull request by mrahim**
First attempt to address #689 

Closes #689

### CODE REVIEW

File: examples/create_geopandas_from_pandas.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""
+Creating a GeoDataFrame from a DataFrame
```
Comment (jorisvandenbossche) [inline examples/create_geopandas_from_pandas.py]: maybe add "with x/y columns" ?

File: examples/create_geopandas_from_pandas.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+###############################################################################
+# A ``GeoDataFrame`` needs a ``shapely`` object, so we create a new column
+# **Coordinates** as a tuple of **Longitude** and **Latitude** :
+
+df['Coordinates']  = list(zip(df.Longitude, df.Latitude))
+
+###############################################################################
+# Then, we transform tuples to ``Point`` :
+
+df['Coordinates'] = df['Coordinates'].apply(Point)
```
Comment (jorisvandenbossche) [inline examples/create_geopandas_from_pandas.py]: I was thinking here a bit about the best way how to do this. 

I think I previously sometimes did:

```
df.apply(lambda row: Point(row.Longitude, row.Latitude), axis=1)
```

but your version is definitely a lot faster, and I think also easier.

File: examples/create_geopandas_from_pandas.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+###############################################################################
+# Now, we can create the ``GeoDataFrame`` by setting ``geometry`` with the
+# coordinates created previously.
+
+gdf = gpd.GeoDataFrame(df, geometry='Coordinates')
+
```
Comment (jorisvandenbossche) [inline examples/create_geopandas_from_pandas.py]: Can you add a `gdf` to show the result? (so we see the column with Points)

**Comment by mrahim** [comment]
Thanks for the feedback, I've added parsing wkt format.

**Pull request merged**